raiden/tests/integration/network/proxies/test_service_registry.py

```diff
@@ -9,6 +9,7 @@
 from raiden.tests.utils.factories import HOP1
 from raiden.tests.utils.smartcontracts import deploy_service_registry_and_set_urls
 from raiden.utils import privatekey_to_address
+from raiden_contracts.constants import CONTRACT_TOKEN_NETWORK_REGISTRY
```

Comment:
The `CONTRACT_TOKEN_NETWORK_REGISTRY` constant is the name of the contract, not the address.

Reply:
oh no my bad, sorry changed it to an address